CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### 💡 Enhancements 💡
+
+- `kafkametricsreceiver`: Migrate kafkametricsreceiver to the mdatagen metrics builder (#7142)
```

Comment:
This changelog line should be migrated to the new process https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#changelog